ACE/ACE-INSTALL.html

```diff
@@ -181,7 +181,7 @@ <h2>Synopsis</h2>
     <td>Maintains ACE on many platforms required for their ACE and
         TAO service offerings. We support AIX,
         Embarcadero C++ Builder,
-        Windows CE, MinGW, Microsoft Visual C++, GCC,
+        MinGW, Microsoft Visual C++, GCC,
         Cygwin, VxWorks 6.x (kernel and rtp), OpenVMS on IA64,
         BlueCAT Linux, RedHat Linux, Fedora, MacOSX, Solaris,
         SuSE Linux on IA32/EM64T/IA64, RTEMS, QNX, LynxOS,
```

Comment:
Out of curiosity, Johnny do still have IA64/Itanium machines that you can build ACE/TAO on?

Reply:
No, we removed them already a few years ago from our build infrastructure, they are very expensive to run and the customers who funded those ports moved to regular hardware